c++/src/Reader.cc

```diff
@@ -286,6 +287,10 @@ namespace orc {
           lastStripe = i + 1;
         }
       }
+      if (footer->rowindexstride() > 0) {
+        fileRowGroupCount +=
```

Comment:
I'd suggest to rename fileRowGroupCount to something like numSplitRowGroups,  numRequestedRowGroups or numRowGroupsInRange. Then refine it to count stripe in range only.

Reply:
Done. The fileRowGroupCount is replaced with numRowGroupsInStripeRange.